config/puma.rb

```diff
@@ -15,6 +15,6 @@
   end
 end
 
-port = Integer(ENV['PORT'] || 3000)
+port = 5000
```

Comment:
🔧 I'd recommend adding documentation for running the app on a different port using a `PORT` setting in the env file. 

📝 Probably also good to check whether the existing `HOST` setting is getting used